wolfssl/ssl.h

```diff
@@ -1057,6 +1063,20 @@ WOLFSSL_API int  wolfSSL_CTX_get_cert_cache_memsize(WOLFSSL_CTX*);
 WOLFSSL_API int  wolfSSL_CTX_set_cipher_list(WOLFSSL_CTX*, const char*);
 WOLFSSL_API int  wolfSSL_set_cipher_list(WOLFSSL*, const char*);
 
+#ifdef HAVE_KEYING_MATERIAL
+#define TLS_PRF_LABEL_CLIENT_FINISHED     "client finished"
```

Comment:
Why are these public?

Reply:
They don't need to be. I'll move them to ssl.c.